### DISCUSSION TITLE
passing actions?

### DISCUSSION THREAD

**Discussion by halindraprakoso** [carbon-design-system/carbon-components-svelte#1347] [Q&A]
hello,

is it possible to pass actions to the TextInput, etc.?

**Comment by tmarsik42** [comment]
You can't. Actions can only be passed to DOM elements, not components. You can work around it by wrapping component in a div.
https://svelte.dev/repl/038586fe6a274d50b15c23b1761fcee2?version=3.49.0

This is more of a svelte question tho.

**Reply by halindraprakoso** [reply]
thanks for the answer. I was attempting to mask an input value so wrapping was not going to work. But I worked around it.
It's not really a svelte question though, cause it's possible to pass down actions like in svelte-material-ui.
